lib/themes/dosomething/paraneue_dosomething/js/components/CampaignSlide.js

```diff
@@ -25,6 +25,7 @@ class CampaignSlide extends React.Component {
 
     this.controller.getCampaigns(args).then(campaigns => {
       this.campaigns = campaigns.splice(0, 3); // Grab the first 3 campaigns returned
+      this.forceUpdate();
```

Comment:
Added this because while testing the campaign slide wasn't updating with campaigns if it was the only slide in the array. I think its safe to keep it